### DISCUSSION TITLE
What happened to the styles folder?

### DISCUSSION THREAD

**Discussion by eduardobragaxz** [microsoft/microsoft-ui-xaml#9051] [General]
https://github.com/microsoft/microsoft-ui-xaml/blob/main/dev/CommonStyles/Button_themeresources.xaml

It seems a recent [change](https://github.com/microsoft/microsoft-ui-xaml/commit/aac7a6ddf1b21a4aa72f3f8eea93745f5ba86f93) deleted this folder and consequently this file, which documentation still links to and it's useful. Should we be looking someplace else for these?

**Comment by kmgallahan** [comment]
https://github.com/microsoft/microsoft-ui-xaml/blob/winui3/release/1.4.2/controls/dev/CommonStyles/Button_themeresources.xaml

**Reply by eduardobragaxz** [reply]
Thanks. Hopefully the url doesn't change every time they update 😐
